spec/models/budget_spec.rb

```diff
@@ -40,6 +42,20 @@
       end
     end
 
+    describe "providing organisation" do
+      context "when the budget_type is a direct type" do
```

Comment:
Do you need a test for the opposite case - i.e. that it **doesn't** get set for non-direct types?

Reply:
Good catch! Incoming updates will include tests for the discarded inputs as well.